### PR TITLE
Expose General Peering Policy from PDB

### PR DESCRIPTION
**Pull request by jamesditrapani**
### Enhancements:
Part of #314, expose General Peering Policy from PDB synced ASNs. This is not a user editable field in Peering Manager and will only be displayed if the ASN is synced with PDB. Driver behind this is a user F/R but also crucial as we now allow users to filter available sessions based on General Policy.

### CODE REVIEW

**Review by gmazoyer — COMMENTED** [review]
To avoid duplicating the data, could we just create a new property returning the policy if the PeeringDB record?

```python
@property
def general_policy(self):
    if self.peeringdb_network:
        return self.peeringdb_network.policy_general
    else:
        return "Unknown"
```

This would save us the trouble of a new field that is already in PeeringDB anyway.

File: peering/models/models.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
     RoutingPolicyType,
 )
 from peering.fields import ASNField, CommunityField
+from peeringdb.enums import GeneralPolicy
```
Comment (gmazoyer) [inline peering/models/models.py:25]: Line can be removed.

**Review by gmazoyer — APPROVED** [review]
LGTM

File: peering/models/models.py (hunk 1 above)
Comment (jamesditrapani) [reply peering/models/models.py]: Sigh, thought isort/black would pick this up :(

**Pull request merged**